test/interval_tests/construction.jl

```diff
@@ -64,7 +64,7 @@ const eeuler = Base.MathConstants.e
     # a < Inf and b > -Inf
     @test @interval(1e300) == Interval(9.999999999999999e299, 1.0e300)
     @test @interval(-1e307) == Interval(-1.0000000000000001e307, -1.0e307)
-    @test @interval(Inf) == IntervalArithmetic.wideinterval(Inf)
```

Comment:
I think we would still want `@interval(Inf)` to return `interval(prevfloat(Inf), Inf)`

And similarly for `-Inf`.

`@interval` is allowed to do special things.

Reply:
okay, so should we comment out tests like these `@test @interval("[+infinity]") == ∅`?

Reply:
If we are deliberately changing the behaviour then we should change the tests to test the new behaviour.

Reply:
> I think we would still want `@interval(Inf)` to return `interval(prevfloat(Inf), Inf)`
> 
> And similarly for `-Inf`.
> 
> `@interval` is allowed to do special things.

Same opinion here. Also, I think that is the expected behavior according to the standard.